modules/ROOT/pages/constraints/managing-constraints.adoc

```diff
@@ -1112,142 +1283,168 @@ RETURN wrote
 
+[[constraint-failures-and-indexes]]
+==== Constraint failures and indexes
 
-.Creating a node property uniqueness constraint on the same name as an existing index
+Attempting to create any type of constraint with the same name as an existing index will fail.
+
+.Creating a node property type constraint with the same name as an existing index
 ======
 
-////
-[source, cypher, role=test-setup]
+.Create an index with the name `directors`
+[source, cypher]
 ----
-CREATE INDEX preExisting_book_word_count FOR (book:Book) ON (book.wordCount)
+CREATE INDEX directors FOR (director:Director) ON (director.name)
 ----
-////
 
-.Create a constraint requiring all `Book` nodes to have unique `wordCount` properties, when an index already exists on that label and property combination
+.Create a node property type constraint with the name `directors`
 [source, cypher, role=test-fail]
 ----
-CREATE CONSTRAINT book_word_count FOR (book:Book) REQUIRE book.wordCount IS UNIQUE
+CREATE CONSTRAINT directors FOR (movie:Movie) REQUIRE movie.director IS :: STRING
 ----
 
-In this case, the constraint cannot be created because there already exists an index covering that schema.
-
 .Error message
 [source, error]
 ----
-There already exists an index (:Book {wordCount}).
-A constraint cannot be created until the index has been dropped.
+There already exists an index called 'directors'.
 ----
 
 ======
 
+Creating key or property uniqueness constraints on the same schema as an existing index will fail.
 
-.Creating a relationship property key constraint on the same name as an existing index
+.Creating a node property uniqueness constraint on the same schema as an existing index
 ======
 
-////
-[source, cypher, role=test-setup]
+.Create an index for `wordCount` properties on `Book` nodes
+[source, cypher]
 ----
-CREATE INDEX knows FOR ()-[know:KNOW]-() ON (know.levels)
+CREATE INDEX book_word_count FOR (book:Book) ON (book.wordCount)
 ----
-////
 
-.Create a constraint requiring all `KNOWS` relationships to have a unique level property as a relationship key, when an index already exists with the given name
+.Create a constraint requiring all `Book` nodes to have unique `wordCount` properties
 [source, cypher, role=test-fail]
 ----
-CREATE CONSTRAINT knows
-FOR ()-[knows:KNOWS]-() REQUIRE (knows.level) IS RELATIONSHIP KEY
+CREATE CONSTRAINT word_count FOR (book:Book) REQUIRE book.wordCount IS UNIQUE
 ----
 
-In this case, the constraint cannot be created because there already exists an index with the given name.
-
 .Error message
 [source, error]
 ----
-There already exists an index called 'knows'.
+There already exists an index (:Book {wordCount}).
+A constraint cannot be created until the index has been dropped.
 ----
 
 ======
 
+
 [[key-and-uniqueness-constraint-on-the-same-schema]]
-==== Key and uniqueness constraints are not allowed on the same schema
+==== Creating key and property uniqueness constraints on the same schema is not allowed
```

Comment:
I guess it lives down here due to it being the index that makes it so they can't coexist, but this is not where I'd look for it... I would have expected it under the `Creating an already existing constraint will fail` since the different property type clashing ones are there... (or have these two in another group of `Creating a constraint conflicting with an already existing constraint will fail` as they are not really "already existing" constraint that you're trying to create)